src/components/ChatList/elements/ChannelImage.style.ts

```diff
@@ -56,7 +56,7 @@ export const channelImageStyles = StyleSheet.create({
     height: dimen.normalizeDimen(24),
     borderRadius: dimen.normalizeDimen(12),
     borderWidth: 2,
-    borderColor: COLORS.white,
+    borderColor: COLORS.almostBlack,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The borderColor has been changed from `COLORS.white` to `COLORS.almostBlack`. This is a significant contrast change and could impact the visibility of the border on different backgrounds. Make sure this change aligns with your design guidelines and doesn't affect the readability or user experience.

```diff
-    borderColor: COLORS.white
+    borderColor: COLORS.almostBlack
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->